packages/components/autocomplete/src/Autocomplete.tsx

```diff
@@ -280,11 +294,13 @@ function _Autocomplete<ItemType>(
             <TextInput
               className={styles.inputField}
               {...inputProps}
-              onFocus={() => {
+              onFocus={(e) => {
                 if (!isOpen) {
                   toggleMenu();
                 }
+                onFocus?.(e as React.FocusEvent<HTMLInputElement>);
               }}
```

Comment:
I'd move it to the beginning of the handler, no need to wait for internal logic
```suggestion
              onFocus={(e) => {
              onFocus?.(e as React.FocusEvent<HTMLInputElement>);
                if (!isOpen) {
                  toggleMenu();
                }
              }}
```

Reply:
I don't think it'll wait since this code is sync, but I moved it anyway